tests/FSharp.Compiler.ComponentTests/TypeChecks/Graph/TrieMappingTests.fs

```diff
@@ -49,22 +49,22 @@ type C = { CX: int; CY: int }
     | current -> Assert.Fail($"mkTrie should always return a TrieNodeInfo.Root, got {current}")
 
     let xNode = trie.Children.["X"]
-    Assert.AreEqual(1, xNode.Children.Count)
+    Assert.Equal(1, xNode.Children.Count)
     Assert.True(Seq.isEmpty xNode.Files)
 
     let yNode = xNode.Children["Y"]
-    Assert.AreEqual(2, yNode.Children.Count)
-    Assert.AreEqual(set [| 2 |], yNode.Files)
+    Assert.Equal(2, yNode.Children.Count)
+    Assert.Equal<Set<FileIndex>>(set [| 2 |], yNode.Files)
```

Comment:
Xunit asserts same types by default and has many implicit overloads for `Assert.Equal` - but not for sets, hence specifying the types explicitly.